### PR TITLE
fix grammatical issues in other_form_controls

### PR DESCRIPTION
**Pull request by yashikabhargava**
### Description
fixed minor grammatical issues in other form controls.

### CODE REVIEW

File: files/en-us/learn/forms/other_form_controls/index.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Play with the interactive example at the top of the {{cssxref("resize")}} refere
 
 ## Drop-down controls
 
-Drop-down controls are a simple way to let users select from many options without taking up much space in the user interface. HTML has two forms of drop down content: the **select box**, and the **autocomplete box**. In both cases the interaction is the same — once the control is activated, the browser displays a list of values the user can select between.
+Drop-down controls are a simple way to let users select from many options without taking up much space in the user interface. HTML has two forms of drop-down content: the **select box**, and the **autocomplete box**. In both cases the interaction is the same — once the control is activated, the browser displays a list of values the user can select between.
```
Comment (dipikabh) [inline files/en-us/learn/forms/other_form_controls/index.md:91]:
```suggestion
Drop-down controls are a simple way to let users select from many options without taking up much space in the user interface. HTML has two types of drop-down controls: the **select box** and the **autocomplete box**. The interaction is the same in both the types of drop-down controls — after the control is activated, the browser displays a list of values the user can select from.
```

File: files/en-us/learn/forms/other_form_controls/index.md [hunk 2]
```diff
@@ -274,7 +274,7 @@ The content inside the {{HTMLElement("progress")}} element is a fallback for bro
 
 A meter bar represents a fixed value in a range delimited by {{htmlattrxref("max","meter")}} and {{htmlattrxref("min","meter")}} values. This value is visually rendered as a bar, and to know how this bar looks, we compare the value to some other set values:
 
-- The {{htmlattrxref("low","meter")}} and {{htmlattrxref("high","meter")}} values divide the range in three parts:
+- The {{htmlattrxref("low","meter")}} and {{htmlattrxref("high","meter")}} values divide the range into three parts:
```
Comment (dipikabh) [inline files/en-us/learn/forms/other_form_controls/index.md:277]:
```suggestion
- The {{htmlattrxref("low","meter")}} and {{htmlattrxref("high","meter")}} values divide the range into the following three parts:
```

File: files/en-us/learn/forms/other_form_controls/index.md [hunk 3]
```diff
@@ -292,7 +292,7 @@ All browsers that implement the {{HTMLElement("meter")}} element use those value
 - If the current value is in the average part of the range, the bar is yellow.
 - If the current value is in the worst part of the range, the bar is red.
 
-Such a bar is created using a {{HTMLElement("meter")}} element. This is for implementing any kind of meter, for example a bar showing total space used on a disk, which turns red when it starts to get full.
+Such a bar is created using a {{HTMLElement("meter")}} element. This is for implementing any kind of meter, for example a bar showing the total space used on a disk, which turns red when it starts to get full.
```
Comment (dipikabh) [inline files/en-us/learn/forms/other_form_controls/index.md:295]:
```suggestion
Such a bar is created by using the {{HTMLElement("meter")}} element. This is for implementing any kind of meter; for example, a bar showing the total space used on a disk, which turns red when it starts to get full.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hello @yashikabhargava, thanks for your PR. I have indicated a few other changes to improve the grammar.

**Review by bsmth — APPROVED** [review]
Thanks both. I'll leave this up to Dipika for a final look before merging 👍🏻

**Review by dipikabh — APPROVED** [review]
Thank you for the fixes!

**Pull request merged**